src/test/java/study/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
```

Comment:
조금 더 직관적 표현이 가능한 `isZero` 메서드를 활용해 보셔도 좋을 것 같아요 😃 
```suggestion
        assertThat(result).isZero();
```